Production_scripts/Process_extracts/extract_care_home.R

```diff
@@ -0,0 +1,123 @@
+#####################################################
+# Care Home Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Social Care Care Home Episodes
+# Description - Match on Client data and Costs data
+#####################################################
+
+
+# Load packages
+library(dplyr)
+library(createslf)
+library(phsmethods)
+library(lubridate)
+
+
+fyyear <- check_year_format("1920")
+
+
+# Read in data---------------------------------------
+
+source_ch_data <- haven::read_sav(get_sc_ch_episodes_path(ext = "zsav")) %>%
+  # select episodes for FY
+  filter(record_keydate1 %in% range(start_fy(fyyear), end_fy(fyyear)) |
+    (record_keydate1 <= end_fy(fyyear) & record_keydate2 >= start_fy(fyyear) | is.na(record_keydate2))) %>%
```

Comment:
`dplyr::range` doesn't work on dates, see `today() %in% range(start_fy("2223"), end_fy("2223"))` or `range(start_fy("2223"), end_fy("2223"))`.

You need to use `>=` & `<=` or [lubridate `%within%`](https://lubridate.tidyverse.org/reference/within-interval.html).

Actually, you could use the function I made on #271 which does exactly that.
```suggestion
  filter(is_date_in_year(record_keydate1, fyyear) |
    (record_keydate1 <= end_fy(fyyear) & record_keydate2 >= start_fy(fyyear) | is.na(record_keydate2))) %>%
```